.github/workflows/build_and_test.yml

```diff
@@ -213,11 +213,11 @@ jobs:
         name: ${{ github.job }}_artifact
         path: ${{ github.job }}.tar.xz
 
-  cmake_u18_i0_xer0_js0_j12:
+  cmake_u20_i0_xer0_js0_j17:
```

Comment:
Looks like these u18 test jobs only run CMake tests. Do we want to run more tests on u20?

Reply:
There are some other jobs like https://github.com/objectcomputing/OpenDDS/runs/8242918577?check_suite_focus=true that run tests on u20.  We can extend test coverage in a future PR.